crates/build-rs/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "build-rs"
+version = "0.2.0"
+rust-version.workspace = true
```

Comment:
Just want to call out that this MSRV seems way too high for this crate.
We may want to lower this in the future.